fbw-common/src/systems/shared/src/navigraph/types.ts

```diff
@@ -1,7 +1,8 @@
 import { Chart } from 'navigraph/charts';
 
 export enum NavigraphSubscriptionStatus {
-  None,
+  Free,
```

Comment:
You need to add corresponding entries to the English localization file in `Settings.ThirdPartyOptions.NavigraphAccountLink.SubscriptionStatus`.